src/util/DeviceFingerprint.js

```diff
@@ -10,11 +10,14 @@
  * See the License for the specific language governing permissions and limitations under the License.
  */
 
-/* eslint complexity: [2, 7] */
+/* eslint complexity: [2, 8] */
 
 define(['vendor/lib/q', 'okta/jquery'], function (Q, $) {
 
   return {
+    _isMessageFromCorrectSource: function($iframe, event) {
```

Comment:
we don't really need to return this function right? We can move it above the `return` statement...
Oh, or are you doing this just for the tests? that is also why you actually added this function instead of just adding the logic inside `onMessageReceivedFromOkta` right?